test/replication.test.js

```diff
@@ -19,1522 +19,1545 @@ describe('Replication / Change APIs', function() {
   var useSinceFilter;
   var tid = 0; // per-test unique id used e.g. to build unique model names
 
-  beforeEach(function() {
-    tid++;
-    useSinceFilter = false;
-    var test = this;
-    dataSource = this.dataSource = loopback.createDataSource({
-      connector: loopback.Memory
-    });
-    SourceModel = this.SourceModel = PersistedModel.extend(
-      'SourceModel-' + tid,
-      { id: { id: true, type: String, defaultFn: 'guid' } },
-      { trackChanges: true });
+  describe('Replication without chunking', function() {
```

Comment:
This is adding too much whitespace changes, can we find a way how to preserve existing tests without changes and only add tests for the new "chunk" mode?